### PR TITLE
Remove payload from AlertItem

### PR DESCRIPTION
**Pull request by mykola-kobets-epam**

### CODE REVIEW

File: api/cloudprotocol/alerts.go [hunk 1]
```diff
@@ -52,22 +52,30 @@ const (
 /***********************************************************************************************************************
  * Types
  **********************************************************************************************************************/
+// AlertItem common alert data.
```
Comment (al1img) [inline api/cloudprotocol/alerts.go:55]: Empty line after header

**Review by MykolaSolyanko — APPROVED** [review]

File: resourcemonitor/resourcemonitor_internal_test.go [hunk 2]
```diff
@@ -1233,43 +1234,13 @@ func (host *testInstancesUsage) FillSystemInfo(instanceID string, instance *inst
 	return nil
 }
 
-func AlertsEqual(alert1, alert2 interface{}) bool {
```
Comment (al1img) [inline resourcemonitor/resourcemonitor_internal_test.go]: Make this commit first and use alertutils directly.

File: utils/alertutils/alertutils.go [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+import (
+	"reflect"
+
+	"github.com/aosedge/aos_common/api/cloudprotocol"
+)
+
+// AlertsPayloadEqual compares alerts ignoring timestamp.
+//
+//nolint:funlen
+func AlertsPayloadEqual(alert1, alert2 interface{}) bool {
```
Comment (al1img) [inline utils/alertutils/alertutils.go:29]: Would be nice to add minimalistic unit test.

**Review by mlohvynenko — APPROVED** [review]

File: api/cloudprotocol/alerts.go (hunk 1 above)
Comment (mykola-kobets-epam) [reply api/cloudprotocol/alerts.go:56]: done

**Pull request merged**